commit_ack.go

```diff
@@ -0,0 +1,8 @@
+package dbft
+
+// CommitAck is an interface for dBFT CommitAck message.
+type CommitAck interface {
+	// Data returns commitAck's data that should be used for the final
+	// block construction.
+	Data() []byte
```

Comment:
Strange one, we expect only signatures to be exchanged during the last phase.

Reply:
Initially I did this because I didn't want to copy-past Commit's interface, but right now with PreCommit  and Commit it's more clear. Commit still has Signature(), and it will hold signature.